apps/backend/src/mutations/TechniqueMutations.spec.ts

```diff
@@ -0,0 +1,117 @@
+import 'reflect-metadata';
+import { container } from 'tsyringe';
+
+import { dummyTechnique1 } from '../datasources/mockups/TechniqueDataSource';
+import {
+  dummyUserOfficerWithRole,
+  dummyUserWithRole,
+} from '../datasources/mockups/UserDataSource';
+import TechniqueMutations from './TechniqueMutations';
+
+const techniqueMutations = container.resolve(TechniqueMutations);
+
+describe('Test technique Mutations', () => {
+  test('A user cannot create a technique', () => {
```

Comment:
May be, we can have an additional test to check the ShortCode(more than 20 characters).

Reply:
Also Technique Name if possible.